text/0000-slatepack.md

```diff
@@ -0,0 +1,391 @@
+
+As a consequence, a `SlatePack` address must be revealed by the party producing a QR code in the `SlatePack Workflow`.
+
+1. Receiver shares `SlatePack Address` via QR
+
+2. Sender scans QR code and the transaction is completed synchronously via Tor by deriving the recipients onion v3 address from their `SlatePack Address`
+
+3. (Fallback) If the synchronous transaction fails, a `SlatePack Message` string is output for manual asynchronous transport by the user
+
+## Implementation Timeline
+
+1. Initial SlatePack implementation introduced with the _July 2020 hard fork_
+  - May or may not support encryption by default yet
+
+2. The proposed SlatePack standard is fully implemented and adopted as a universal transaction standard in _last hard fork (Jan 2021)_
+  - SlatePack is the default transaction standard in all wallets and services
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+- This puts a lot of eggs in one basket (if SlatePack fails there will likely be confusion returning to old methods)
+
+- This may be a bit rushed to have where we want it before HF schedule
+
+- Deprecating HTTP(S) is already a major change- by requiring the adoption of this completely new standard in addition we risk putting a lot of effort on the shoulders of existing services in the Grin ecosystem
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+- By adding new options without simplifying the workflow for users we risk confusion and friction
+
+- We could just add an option for copy and pastable slates instead of introducing an entirely new universal transaction flow standard
+
+# Prior art
+[prior-art]: #prior-art
+
+## Tari
+
+Tari handles this general problem by...
+
+## Beam
+
+Beam handles this problem by...
+
+# Unresolved questions
+[unresolved-questions]: #unresolved-questions
+
+- How to handle key derivation harmoniously?
+
+- What are unmentioned security considerations for using the same base key to both map to an onion address and map to an encryption key used in transactions?
+  - Related, what are unmentioned security considerations to `SlatePack Address` reuse?
+
+- What should the `human-readable part` of the bech32 encoded `SlatePack Address` be?
+  - `slatepack1p4fuklglxqsgg602hu4c4jl4aunu5tynyf4lkg96ezh3jefzpy6swshp5x`
+  - `grin1p4fuklglxqsgg602hu4c4jl4aunu5tynyf4lkg96ezh3jefzpy6swshp5x`
+  - `spk1p4fuklglxqsgg602hu4c4jl4aunu5tynyf4lkg96ezh3jefzpy6swshp5x`
+  - `sp1p4fuklglxqsgg602hu4c4jl4aunu5tynyf4lkg96ezh3jefzpy6swshp5x`
+  - `g1p4fuklglxqsgg602hu4c4jl4aunu5tynyf4lkg96ezh3jefzpy6swshp5x`
+
+# Future possibilities
+[future-possibilities]: #future-possibilities
+
+- Extended to support future modes (payment channel, payjoin etc)
+
+- An entirely different standard could be adopted in the future if interactive transactions become the default, eliminating the need for SlatePack
```

Comment:
Interactive transactions are already the default. Do you just mean synchronous here?

Reply:
That was a typo, it should be "non-interactive transactions"